src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
@rnveach , do make sense.
Non-Ttight-Html is like non compilable java file. TreeWalker throw error and stop processing.
Unfortunately javadoc is never correct, so we need an option in TreeWalker like `violateNonTightJavadocHtml` with TRUE by default, if TRUE - TreeWalker will raise single violation on javadoc. Checks will decide by their setting to run on non-tight-html or not. There will be always checks that do not validate javadoc parts that could have HTML, or they simply remove all tags and keep only their text.

@Vladlis , @rnveach , @ps-sp , please share your ideas about this proposal.